src/webui/layouts/application/Application.tsx

```diff
@@ -10,18 +10,18 @@ export type ApplicationProps = {
     main: React.ReactNode;
 };
 
-export default function Application(props: ApplicationProps)
+export default function Application({ aside, main }: ApplicationProps)
```

Comment:
The name of the function should be `Layout` to be consistent with the rest of the pattern for webui implementations